Octokit/Clients/OrganizationMembersClient.cs

```diff
@@ -37,6 +37,22 @@ public enum OrganizationMembersRole
         Member
     }
 
+    public enum OrganizationMembershipsState
+    {
+        [Parameter(Value = "active")]
+        Active,
+        [Parameter(Value = "pending")]
+        Pending
+    }
+    
+    public enum OrganizationMembershipsRole
```

Comment:
I'm not 100% sold on the naming of this `enum`...we now have 3 quite similarly named `enum`s:
- This newly added one
- [OrganizationMembersRole](https://github.com/octokit/octokit.net/blob/9bb344cd19d0b12024c534c0a3accc0467f7abd4/Octokit/Clients/OrganizationMembersClient.cs#L28) - this has the same roles as the one I've added, but with an extra value `All`
- [OrganizationMembershipRole](https://github.com/octokit/octokit.net/blob/7c170213fd2b2a803af3ea38e2ebe610ab738173/Octokit/Models/Response/OrganizationMembershipRole.cs#L5) - this one has **very** different values

Reply:
Now that I found the `MembershipState` `enum`, I'm thinking that `MembershipRole` is a better name for this `enum`